core/src/components/modal/gestures/sheet.ts

```diff
@@ -323,6 +346,13 @@ export const createSheetGesture = (
         },
       ]);
 
+      if (contentAnimation) {
+        contentAnimation.keyframes([
+          { offset: 0, maxHeight: `${(1 - breakpointOffset) * 100}%` },
+          { offset: 1, maxHeight: `${snapToBreakpoint * 100}%` },
+        ]);
```

Comment:
Let's add a comment of what this block of code is doing. [Example](https://github.com/ionic-team/ionic-framework/blob/9794a63f85ca81103d3bbb1a166a81550ba442f4/core/src/components/modal/gestures/sheet.ts#L345-L350) can be found.